rllib/tuned_examples/simple_q/cartpole-simpleq-test.yaml

```diff
@@ -1,4 +1,3 @@
-# config for quick cli testing. does not give meaningful results.
```

Comment:
Hmm, True, though, should we make this example not learn to the end (e.g. 150 reward)? It's pretty quick with SimpleQ. Otherwise, it's not really a "tuned" example :)

Reply:
yeah, good point. maybe we should move these files to another folder or so? In this case you're absolutely right, but I really want to have a couple of test files that return in a manner of seconds, not minutes - and give me a checkpoint to evaluate.

Reply:
@sven1977 ok, I reset the reward for these examples back to 150, makes sense. if we need other, quicker configs, we can add them later.